funnel/views/project.py

```diff
@@ -128,13 +130,15 @@ def project_edit(profile, project):
         form = ProposalSubprojectForm(obj=project, model=Project)
     else:
         form = ProjectForm(obj=project, model=Project)
+    del form.name
```

Comment:
Remove this. We'll do a clean fix for editing `name`, not this hack.

Reply:
Fixed the form validation for name.